app/src/main/java/org/jellyfin/androidtv/ui/browsing/StdGridFragment.java

```diff
@@ -496,48 +495,43 @@ public void onClick(View v) {
     private JumplistPopup mJumplistPopup;
     class JumplistPopup {
 
-        final int WIDTH = Utils.convertDpToPixel(TvApp.getApplication(), 900);
-        final int HEIGHT = Utils.convertDpToPixel(TvApp.getApplication(), 55);
+        private final int WIDTH = Utils.convertDpToPixel(requireContext(), 900);
+        private final int HEIGHT = Utils.convertDpToPixel(requireContext(), 55);
 
-        PopupWindow mPopup;
-        Activity mActivity;
-        JumpList mJumplist;
+        private final PopupWindow popupWindow;
+        private final AlphaPicker alphaPicker;
 
-        JumplistPopup(Activity activity) {
-            mActivity = activity;
-            LayoutInflater inflater = (LayoutInflater) activity.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
+        JumplistPopup() {
+            LayoutInflater inflater = (LayoutInflater) requireActivity().getSystemService(Context.LAYOUT_INFLATER_SERVICE);
             View layout = inflater.inflate(R.layout.empty_popup, null);
```

Comment:
```suggestion
            View layout = inflater.inflate(R.layout.empty_popup, mGridDock, false);
```

I'm not exactly sure if mgriddock is the parent, but it's used in the `show` function